apps/prairielearn/src/pages/partials/questionAwardedPoints.ejs

```diff
@@ -4,25 +4,32 @@
 
 <% if (previous_variants) { %>
     <%
-        const collapseId = `variants-points-collapse-${instance_question_id}`;
+        const collapseClass = `variants-points-collapse-${instance_question_id}`;
         const collapseButtonId = `variants-points-collapse-button-${instance_question_id}`;
-        let startHiddenIndex = null;
-        let endHiddenIndex = null;
     %>
     <% if (previous_variants.length > MAX_DISPLAYED_VARIANTS) { %>
-        <a id="<%= collapseButtonId %>" role="button" onclick="$(`#<%= collapseId %>`).css('display','inline'); $(`#<%= collapseButtonId %>`).css('display','none');">&hellip;</a>
-        <% startHiddenIndex = 0; %>
-        <% endHiddenIndex = previous_variants.length - MAX_DISPLAYED_VARIANTS; %>
+        <a
+            id="<%= collapseButtonId %>"
+            role="button"
+            onclick="
+                // show all the hidden variant score buttons
+                document.querySelectorAll('.<%= collapseClass %>').forEach(e => e.style.display = 'inline-block');
```

Comment:
Nit: the usual suggestion I see in this case to set display to `''` (blank), so it takes the default display of the context.

Reply:
Oh nice! TIL! Changed in 78fe96881285365ac201672163f65482e41d1ddc